### ISSUE TITLE
Importing eventio fails with new numpy version

### ISSUE THREAD

**Issue by pawel21** [cta-observatory/pyeventio#226]
Hi,

I have pulled the last revision of ctapipe_io_lst, after I have executed `pip install -e .`, then I got  "Successfully installed ctapipe-0.10.3 ctapipe-io-lst eventio-1.5.1 setuptools-scm-5.0.1 traitlets-5.0.5".
When I tried import `from ctapipe_io_lst import LSTEventSource ` or `from ctapipe.io import event_source` , 
I got error:
```
...
~/anaconda3/envs/lst-dev-ctapipe0.8/lib/python3.7/site-packages/eventio/tools.py in <module>
      1 import struct
      2 import numpy as np
----> 3 from .var_int import get_length_of_varint, parse_varint
      4 
      5 

eventio/var_int.pyx in init eventio.var_int()

ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 88 from C header, got 80 from PyObject
```

**Comment by maxnoe** [comment]
@pawel21 Could you try:

```
pip uninstall eventio
pip install --no-cache eventio
```
?

**Comment by pawel21** [comment]
> @pawel21 Could you try:
> 
> ```
> pip uninstall eventio
> pip install --no-cache eventio
> ```
> 
> ?

Still the same problem.
After pip install, I got:
```
ERROR: lstchain 0.5.2.post463+git745c824 has requirement ctapipe~=0.8.0, but you'll have ctapipe 0.10.3 which is incompatible.
ERROR: lstchain 0.5.2.post463+git745c824 has requirement gammapy>=0.18, but you'll have gammapy 0.17 which is incompatible.
Installing collected packages: eventio
Successfully installed eventio-1.5.1
```

**Comment by maxnoe** [comment]
Ah, the problem is that the build happens with numpy 1.20 but then in your env you have numpy 1.19.

Try updating numpy to 1.20 in the environment:

```
conda install numpy=1.20
```

**Comment by maxnoe** [comment]
@pawel21 Did you update your lst environment? That should automatically fix all the issues.

So just run:

```
conda env update -f environment.yml
```

**Comment by pawel21** [comment]
> @pawel21 Did you update your lst environment? That should automatically fix all the issues.
> 
> So just run:
> 
> ```
> conda env update -f environment.yml
> ```

Thank you, this problem is solved.
Now, I have problem with `ctapipe_io_lst`, so I will put this problem in `ctapipe_io_lst`

**Comment by pawel21** [comment]
Thank you, problem with `numpy` is solved, but now when I import `from ctapipe_io_lst import LSTEventSource`
I got error:
```
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-1-e35803209b7c> in <module>
----> 1 from ctapipe_io_lst import LSTEventSource

~/Pulpit/Astrophysics/CTA/soft/ctapipe_io_lst/ctapipe_io_lst/__init__.py in <module>
     28 from .version import __version__
     29 from .calibration import LSTR0Corrections
---> 30 from .event_time import EventTimeCalculator
     31 from .pointing import PointingSource
     32 from .anyarray_dtypes import (

~/Pulpit/Astrophysics/CTA/soft/ctapipe_io_lst/ctapipe_io_lst/event_time.py in <module>
     13 if astropy.version.major == 4 and astropy.version.minor <= 2 and astropy.version.bugfix <= 0:
     14     # clear the cache to not depend on import orders
---> 15     TimeFromEpoch.__dict__['_epoch']._cache.clear()
     16     # fix for astropy #11245, epoch was wrong by 8 seconds
     17     TimeUnixTai.epoch_val = '1970-01-01 00:00:00.0'

KeyError: '_epoch'
```
I am working with master branch.

**Comment by maxnoe** [comment]
What is your astropy version?

**Comment by pawel21** [comment]
> What is your astropy version?

4.1

**Comment by maxnoe** [comment]
Update to 4.1 is not supported anymore (by astropy) you should update to 4.2.

However, I will also fix this for astropy 4.1.
